ansible/roles/debops.discourse/defaults/main.yml

```diff
@@ -0,0 +1,686 @@
+# .. envvar:: discourse_app_root_path [[[
+#
+# Directory where the instance of the application gets installed.
+discourse_app_root_path: '{{ (ansible_local.root.app
+                           if (ansible_local|d() and ansible_local.root|d() and
+                               ansible_local.root.app|d())
+                           else "/var/local") + "/" + discourse_user + "/" + discourse__instance_name }}'
+
+                                                                   # ]]]
+# .. envvar:: discourse_backup_path [[[
+#
+# The path where backups are stored for this instance.
+discourse_backup_path: '{{ (ansible_local.root.backup
+                         if (ansible_local|d() and ansible_local.root|d() and
+                             ansible_local.root.backup|d())
+                         else "/var/backups") + "/discourse/{{discourse__instance_name}}" }}'
+
+                                                                   # ]]]
+# .. envvar:: discourse_src_path [[[
+#
+# Discourse sources root path. The source is unpacked resp. cloned
+# here from the remote location.
+discourse_src_path: '{{ (ansible_local.root.src
+                      if (ansible_local|d() and ansible_local.root|d() and
+                          ansible_local.root.src|d())
+                      else "/usr/local/src") + "/discourse" }}'
+
+                                                                   # ]]]
+                                                                   # ]]]
+# System user, group, additional groups [[[
+# -----------------------------------------
+
+# .. envvar:: discourse_user [[[
+#
+# System UNIX account used by the Discourse application. It will be visible in the
+# :command:`git+ssh` remote URLs.
+discourse_user: '{{discourse__instance_name}}'
+
+                                                                   # ]]]
+# .. envvar:: discourse_group [[[
+#
+# System UNIX group used by the Discourse application.
+discourse_group: '{{discourse__instance_name}}'
+
+                                                                   # ]]]
+# .. envvar:: discourse_user_append_groups [[[
+#
+# List of additional system groups to add to the Discourse user account.
+# The ``sshusers`` UNIX group is used in DebOps to limit SSH access. See the
+# :ref:`debops.system_groups` role for more details.
+discourse_user_append_groups: []
+
+                                                                   # ]]]
+# .. envvar:: discourse__shell [[[
+#
+# The default shell used by the Discourse UNIX account.
+discourse__shell: '/bin/bash'
+                                                                   # ]]]
+                                                                   # ]]]
+# Internal application options [[[
+# --------------------------------
+
+# .. envvar:: discourse_unicorn_workers [[[
+#
+# Number of Unicorn workers to start.
+# If you only have 1GB RAM, use 2 workers. Otherwise increase to 3 or 4.
+discourse_unicorn_workers: 2
```

Comment:
How about using `ansible_processor_vcpus` as default here? It would simplify scaling - just set up a bigger VM or use more powerful host.